webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable no-unused-vars */
+// eslint bug - https://github.com/eslint/eslint/issues/12117
```

Comment:
Some developers suggested using `eslint v6.7.1` and some suggest to update `babel-eslint` to `10.0.3`

Reply:
doesn't work :/
```
/home/afeferku/git/foreman/webpack/assets/javascripts/react_app/components/Layout/LayoutHelper.js
  13:14  error  'item' is defined but never used   no-unused-vars
  14:16  error  'child' is defined but never used  no-unused-vars
```